tests/layer_tests/tensorflow_tests/test_tf_ArgMinMax.py

```diff
@@ -0,0 +1,70 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+
+from common.tf_layer_test_class import CommonTFLayerTest
+
+# Testing operation ArgMin, ArgMax (Initial Implementation)
+# Documentation: https://www.tensorflow.org/api_docs/python/tf/raw_ops/ArgMin
+#                https://www.tensorflow.org/api_docs/python/tf/raw_ops/ArgMax
+
+class TestArgMinMax(CommonTFLayerTest):
+    # input_shape - should be an array
+    # dimension - dimension to be used, for vector should be 0
+    # op_type - type of testing operation
+    # ir_version - common parameter
+    # use_new_frontend - common parameter
+    def create_argminmax_placeholder_const_net(self, input_shape, dimension, op_type, ir_version, use_new_frontend):
+        """
+            Tensorflow net                  IR net
+
+            Placeholder->op_type    =>       Placeholder->TopK->Squeeze
+                         /                               /     /
+            Const-------/                   Const-------/-----/
+
+        """
+
+        import tensorflow as tf
+
+        tf.compat.v1.reset_default_graph()
+
+        # Create the graph and model
+        with tf.compat.v1.Session() as sess:
+            op_type_to_tf = {
+                'ArgMax': tf.raw_ops.ArgMax,
+                'ArgMin': tf.raw_ops.ArgMin,
+            }
+            tf_input_shape = input_shape.copy()
```

Comment:
please do not copy shape and pass it directly.

Reply:
We should cleanup old tests too, otherwise it looks like confusing (even for me): why some tests use copies and premute, but others don't.
What do you think?